app/main.py

```diff
@@ -1,12 +1,42 @@
+import os
+import asyncio
 from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from dotenv import load_dotenv
+from langchain_openai import OpenAI
+from langchain.chains import LLMChain
+from langchain.prompts import PromptTemplate
+
+load_dotenv()  # Load environment variables from .env file
+LLM_API_KEY = os.getenv('LLM_API_KEY')
 
 app = FastAPI()
 
+# Set up LangChain with OpenAI
```

Comment:
Can we move this class to a different file? We can default OpenAI agent as default agent until we support local models